### PR TITLE
Implement ember paper

### PR DESCRIPTION
**Pull request by adiehl96**
Contains components that are used in [#812](https://github.com/Fabriquartz/resource-manager/pull/812) will probably be needed to test that pr

### CODE REVIEW

File: addon/components/auto-complete.js [hunk 1]
```diff
@@ -61,6 +61,7 @@ export default Component.extend(formFieldOptions, {
   actions: {
     update(value) {
       set(this, '_selectAll', false);
+      console.log("value in autocomp", value)
```
Comment (cbroeren) [inline addon/components/auto-complete.js]: Whoops

File: tests/dummy/app/templates/components/test-component.hbs [hunk 2]
```diff
@@ -27,3 +27,17 @@
...
+             object=object
+             propertyPath="text"
+             updated=(action "update")
+             disabled=true
+             label="lel"}}
+<br/>
+
+
+<div class="layout layout-sm-column">
+{{paper-input class="flex-30" label="Name" value=object.text onChange=(action (mut object.text)) placeholder="Address" disabled=true}}
```
Comment (cbroeren) [inline tests/dummy/app/templates/components/test-component.hbs]: Use newlines

File: addon/templates/components/form-paper.hbs [hunk 3]
```diff
@@ -0,0 +1,7 @@
+{{log "inside-from-paper" value}}
```
Comment (cbroeren) [inline addon/templates/components/form-paper.hbs]: Whoops

File: addon/components/form-paper.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import Component        from 'ember-component';
+import textInputMixin   from 'ember-railio-form-components/mixins/text-input-mixin';
+import formFieldOptions from
+  'ember-railio-form-components/mixins/form-field-options';
+
+import layout from '../templates/components/form-paper';
+
+
```
Comment (cbroeren) [inline addon/components/form-paper.js]: 2 lines

File: addon/components/form-paper.js [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+export default Component.extend(textInputMixin, formFieldOptions, {
+  classNames: ['text-field'],
```
Comment (cbroeren) [inline addon/components/form-paper.js]: When this is a text-field, form-paper is not a right name. I prefer paper-form-field anyway

File: app/templates/components/form-field.hbs [hunk 6]
```diff
@@ -1,15 +1,17 @@
-{{#if labelText}}
+{{#if (and labelText (not isPaper))}}
   <label>{{labelText}}</label>
 {{/if}}
 
 {{#if hasBlock}}
   {{yield (get object propertyPath) (action "update") disabled}}
 {{else}}
+  {{log type}}
```
Comment (cbroeren) [inline app/templates/components/form-field.hbs]: Whoops

File: app/templates/components/form-field.hbs [hunk 7]
```diff
@@ -1,15 +1,17 @@
...
   {{component type value=(get object propertyPath)
-                   name=name
-                   componentProperties=options
-                   updated=(action "update")
-                   disabled=disabled}}
+                  input=input
```
Comment (cbroeren) [inline app/templates/components/form-field.hbs]: What is this input?

Comment (cbroeren) [inline app/templates/components/form-field.hbs]: Add space back, to align with the value

File: tests/dummy/app/templates/components/test-component.hbs [hunk 8]
```diff
@@ -27,3 +27,17 @@
...
+{{form-field type="paper-input"
+             object=object
+             propertyPath="text"
+             updated=(action "update")
+             disabled=true
+             label="lel"}}
+<br/>
+
+
+<div class="layout layout-sm-column">
```
Comment (cbroeren) [inline tests/dummy/app/templates/components/test-component.hbs]: Why did you add this div and it's content. It's a direct paper input. Nothing to do with our form-fields right?

**Comment by adiehl96** [comment]
This was indeed not a final version, I should have had the WIP label assigned
